source/manuals/programming-languages/python/python.html.md.erb

```diff
@@ -171,6 +171,15 @@ Note: you can also ignore rules on particular lines of code or files by adding a
 * [Flake8 plugins list][flake8-plugins]
 
 
+## Environments
+
+Use distinct Python environments for each project you work on.
```

Comment:
Is it worth explaining here what is meant by 'environment'? For instance, I'm not sure on whether we're recommending the use of `pyenv-virtualenv` as well as `pyenv`.

Reply:
I can see your point, but we do assume a level of knowledge. Perhaps pointing to https://realpython.com/python-virtual-environments-a-primer/ would help

Reply:
Fair. Maybe it would just be helpful to add `virtual` to it.

```suggestion
Use distinct Python virtual environments for each project you work on.
```

I think my confusion kind of happened because we say in the following line that 'we recommend Pyenv to manage Python environments', but `pyenv` on its own doesn't manage virtualenvs.

Reply:
yeah absolutely, i'm working on clarifying the language here around python versions and virtual envs and making that distinction